lib/rubocop/config_formatter.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+module RuboCop
+  # Builds YAML config file from two hashes
+  class ConfigFormatter
+    NAMESPACES =
```

Comment:
I wonder whether this can be generated from `Cop.registry.departments`

Reply:
Thanks! Done.